Makefile

```diff
@@ -44,7 +47,7 @@ build-examples: $(EXAMPLES_BINARIES)
 
 $(EXAMPLES_BINARIES): $(BUILD_DIR)/%: $(SOURCE_FILES) $(EXAMPLES_SOURCE_FILES) | $(BUILD_DIR)
 	$(GET_DEPENDENCIES_WITH)
-	$(PONYC) -o $(BUILD_DIR) $(EXAMPLES_DIR)/$*
+	$(PONYC) -o $(BUILD_DIR) $(EXAMPLES_DIR)/$(*:.exe=)
```

Comment:
I think I would prefer the following if it works on Windows:
```make
...
EXE_EXT ?= $(if $(WINDIR),.exe,)
...
tests_binary := $(BUILD_DIR)/$(PACKAGE)$(EXE_EXT)
...
EXAMPLES_BINARIES := $(addsuffix $(EXE_EXT),$(addprefix ${BUILD_DIR}/,${EXAMPLES}))
```
Then the pattern for the `EXAMPLES_BINARIES` target can be `$(BUILD_DIR)/%$(EXE_EXT)` and `$*` will be the target name as expected

Reply:
Yep this works great. Will pull in now.